src/backend.c

```diff
@@ -78,7 +82,7 @@ static int backend_ctx_find_alias_devices (hashcat_ctx_t *hashcat_ctx)
       {
         if (device_param_dst->skipped == false)
         {
-          device_param_dst->skipped = true;
+          device_param_src->skipped = true;
```

Comment:
Please explain switch to device_param_src?

Reply:
because with APPLE the right device is skipped instead of get selected

Reply:
OK, but will not be needed if fixed based on vendor_id is used.